app/boards/arm/bdn9/bdn9_rev2.yaml

```diff
@@ -1,5 +1,5 @@
-identifier: bdn9
-name: keeb.io BDN9
+identifier: bdn9_rev2
```

Comment:
It's likely some numbnut will submit a PR where the identifier doesn't match the filenames.  I suggest we pre-empt that with a GitHub workflow/job that protects against it.  Or at least create an issue to do so or we'll forget.